wps_shared/wps_shared/db/crud/fuel_layer.py

```diff
@@ -0,0 +1,17 @@
+"""CRUD operations relating to processing snow coverage"""
```

Comment:
copy pasta on 'snow coverage`

Reply:
Thanks, done in: https://github.com/bcgov/wps/pull/4428/commits/fa46b23068257d4a9415e0cc9b9610648e8e1e92